server/src/main/java/org/opensearch/Version.java

```diff
@@ -287,7 +287,7 @@ public Version minimumCompatibilityVersion() {
 
     protected Version computeMinCompatVersion() {
         if (major == 1) {
-            return Version.fromId(6080099);
+            return Version.fromId(6070099);
```

Comment:
@nknize we can keep it as 6.8.0. Changing this here would break a test InboundDecoder

Reply:
Yeah.. it also breaks `SimpleMockNioTransportTests`